src/components/SignName.vue

```diff
@@ -103,7 +112,8 @@ export default {
     name: 'SignName',
     data: () => ({
       errorMessages: '',
-      name: null,
+      firstName: null,
```

Comment:
Technically this works, but I think we should prefer an empty string for uninitialized string fields. That will still be a falsy value and it prevents certain exceptions where a function might expect a string.

Reply:
Updated uninitialized string fields as empty strings